docs/learning/Capabilities.md

```diff
@@ -186,7 +186,7 @@ Assistants for Tools to find existing geometry and for creating new geometry:
 
 Load new functionality to a running instance of an application in a web browser.
 
-- 👷 Load registered [Extension](./frontend/Extensions.md)
+- 👷 Load registered `Extension`
```

Comment:
Might want to remove this section altogether and add it back later?

Reply:
Probably a good idea, I was reading through this entire doc and wanted to make a bunch of changes to it...  I'll just make those changes for now.

Reply:
pushed an update